life_expectancy/cleaning.py

```diff
@@ -0,0 +1,66 @@
+"""
+Assignment 1
+"""
+import argparse
+import numpy as np
+import pandas as pd
+
+def load_data():
+    """
+    load data from file
+    """
+    data = pd.read_csv('life_expectancy/data/eu_life_expectancy_raw.tsv', sep='\t')
+    return data
+
+def save_data(data):
+    """
+    save data to file
+    """
+    data.to_csv('life_expectancy/data/pt_life_expectancy.csv', index = False)
+
+def clean_data(data, country = 'PT'):
+    """
+    Cleans data.
+    """
+    life_expectancy = data
+
+    id_variables = ['unit','sex','age','region']
+
+    life_expectancy[id_variables] = (
+        life_expectancy['unit,sex,age,geo\\time'].str.split(',', expand = True)
+    )
+    life_expectancy = life_expectancy.drop(columns=['unit,sex,age,geo\\time'])
+    life_expectancy = life_expectancy[life_expectancy["region"] == country]
```

Comment:
If case is not correct, you might miss some filters
```suggestion
    life_expectancy = life_expectancy[life_expectancy["region"].str.upper() == country.str.upper()]
```